content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__class__.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
@tinaafitz and regarding 4) the state will be part of the ImportVm state machine (after the last "post" state)?